extension-inspector-analyser/src/main/java/org/alfresco/extension_inspector/analyser/result/JakartaMigrationConflict.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.alfresco.extension_inspector.analyser.result;
+
+import static org.alfresco.extension_inspector.analyser.result.Conflict.Type.JAKARTA_MIGRATION_CONFLICT;
+
+import java.util.Objects;
+import java.util.Set;
+
+import org.alfresco.extension_inspector.model.ClasspathElementResource;
+
+public class JakartaMigrationConflict extends AbstractConflict
+{
+    private Set<String> invalidJakartaMigrationDependencies;
+
+    public JakartaMigrationConflict()
+    {
+    }
+
+    public JakartaMigrationConflict(ClasspathElementResource ampResourceInConflict,
+                                         Set<String> invalidJakartaMigrationDependencies, String alfrescoVersion)
+    {
+        super(JAKARTA_MIGRATION_CONFLICT, ampResourceInConflict, null, alfrescoVersion);
+        this.invalidJakartaMigrationDependencies = invalidJakartaMigrationDependencies;
+    }
+
+    public Set<String> getInvalidJakartaMigrationDependencies()
+    {
+        return invalidJakartaMigrationDependencies;
+    }
+
+    public void setInvalidJakartaMigrationDependencies(Set<String> invalidJakartaMigrationDependencies)
```

Comment:
The `setInvalidJakartaMigrationDependencies` method is never used. It can be removed and the `invalidJakartaMigrationDependencies` field can be made `final`.